site/src/pages/components/selectlist.mdx

```diff
@@ -52,6 +52,7 @@ Because the various parts of the selectlist can be styled, it's important to und
 - `<selectedvalue>` - The element which contains the text of the currently selected option. Every time that the user selects an option, the browser will replace the text content of this element with the text content of the selected option.
 - `<listbox>` - The wrapper that contains the `<option>`(s) and `<optgroup>`(s). If this part was not provided by the author, then `<selectlist>` will automatically create one.
 - `<optgroup>` - Optional element which groups `<option>`(s) together with a label.
+- `<legend>` - Used to provide a label to an `<optgroup>` element. It must be the first child of the `<optgroup>`.
```

Comment:
Must it be the first child? I know it's likely bad to have more than one of them, but I wonder if it's ok to put it anywhere you like?

Reply:
MUST it be from a browser implementation pov?  i mean, probably not?  i'll defer to you on that.  automagical stuff with details > summary, fieldset > legend, table > caption and how those are treated (re-positioned) if not the first items vs how they're rendered if not the first instance of the specific elements.

from an author / introduce the thing you're grouping with the group label, rather than randomly have that in the middle of your group - thus making it visually awkward to where the group beings POV... i think it's a pretty darn good author must, unless there's going to be automagical re-positioning of this element similar to the others?

Reply:
from another angle, even for instances where maybe someone wants to be a bit more creative with how they're laying out their label for a group of options, shouldn't this be possible with modern CSS to visually reposition without needing to also futz with the actual DOM ordering?